assets/js/cookies.js

```diff
@@ -0,0 +1,17 @@
+const displayCookies = () => {
+  const cookiesBanner = document.getElementById("cookies-banner");
+
+  if (sessionStorage.getItem("cookies-accept")) {
```

Comment:
I'm wondering if we should use the sessionStorage or the localStorage for this. With the session storage it would show up everytime you close and reopen the browser (i.e. the session finished). 

Or we might use localStorage and set an expiry date. Apparently you cannot set an expiry date directly, but I found this: https://www.sohamkamani.com/blog/javascript-localstorage-with-ttl-expiry/

Howvever, I'm happy to hear other opinions on that. Why did you use sessionStorage over localStorage in the first place?